### PR TITLE
Collection of small admin area changes

### PR DESCRIPTION
**Pull request by Boy132**
Many little things that were left todo in the admin area.

- Enabled the tag inputs for nodes
- Updated the icon of the cpu column for nodes
- Disabled `inline` for `Force outgoing ip` toggle
- Changed label for `Database Hosts`
- Added custom empty states for `Database Hosts` & `API Keys`
- Added Icons to the tabs on the EditEgg page
- Fixed a small typo in `ServersRelationManager`
- Renamed Node `Automatic Allocation` (`public` field) to avoid confusion with the `Automatic Allocations` (allow clients to create allocations via the frontend)
- Ran code cleanup (mainly reordered imports or removed unnecessary code)
- Removed regex for node display name

### CODE REVIEW

File: app/Filament/Resources/DatabaseHostResource.php [hunk 1]
```diff
@@ -10,7 +10,7 @@ class DatabaseHostResource extends Resource
 {
     protected static ?string $model = DatabaseHost::class;
 
-    protected static ?string $label = 'Databases';
+    protected static ?string $label = 'Database Host';
```
Comment (lancepioch) [inline app/Filament/Resources/DatabaseHostResource.php:13]: I'm not saying this is wrong, but this is what Pterodactyl had before. Just food for thought.

**Review by lancepioch — APPROVED** [review]
Looks good, awesome!

File: app/Filament/Resources/ApiKeyResource/Pages/ListApiKeys.php [hunk 2]
```diff
@@ -51,13 +52,23 @@ public function table(Table $table): Table
...
             ]);
     }
 
     protected function getHeaderActions(): array
     {
         return [
-            Actions\CreateAction::make(),
+            Actions\CreateAction::make()
+                ->label('New API Key')
+                ->hidden(fn () => ApiKey::count() <= 0),
```
Comment (RMartinOscar) [inline app/Filament/Resources/ApiKeyResource/Pages/ListApiKeys.php:71]: Why would you hide it if there's no key yet ?

Comment (notAreYouScared) [reply app/Filament/Resources/ApiKeyResource/Pages/ListApiKeys.php:71]: It's shown in the empty state

File: app/Filament/Resources/DatabaseHostResource.php (hunk 1 above)
Comment (notAreYouScared) [reply app/Filament/Resources/DatabaseHostResource.php:13]: I think for clarification, Database Hosts is correct. It's referred to that in discord when people talk and on the docs

File: app/Filament/Resources/ApiKeyResource/Pages/ListApiKeys.php (hunk 2 above)
Comment (notAreYouScared) [inline app/Filament/Resources/ApiKeyResource/Pages/ListApiKeys.php:71]: image
You are checking for all API keys, this also includes user API keys, resulting in both buttons appearing if a client has an API key, we need to change for `key_type`  = `2` 

1 = User
2 = Application

File: app/Filament/Resources/ApiKeyResource/Pages/ListApiKeys.php [hunk 3]
```diff
@@ -51,13 +52,23 @@ public function table(Table $table): Table
...
+                    ->button(),
             ]);
     }
 
     protected function getHeaderActions(): array
     {
         return [
-            Actions\CreateAction::make(),
+            Actions\CreateAction::make()
+                ->label('New API Key')
```
Comment (notAreYouScared) [inline app/Filament/Resources/ApiKeyResource/Pages/ListApiKeys.php:70]: Create? We do not use "New" anywhere else in the panel.

File: app/Filament/Resources/DatabaseHostResource.php (hunk 1 above)
Comment (notAreYouScared) [inline app/Filament/Resources/DatabaseHostResource.php:13]: image
With this change, you also need to edit the sidebar width.

**Review by notAreYouScared — APPROVED** [review]
LGFMH

**Pull request merged**